crates/bevy_input/src/input.rs

```diff
@@ -35,7 +35,7 @@ use bevy_ecs::schedule::State;
 /// * Call the [`Input::release`] method for each release event.
 /// * Call the [`Input::clear`] method at each frame start, before processing events.
 #[derive(Debug, Clone, Resource, Reflect)]
-#[reflect_value]
+#[reflect_value(Default)]
```

Comment:
I might have already asked this, but does this need to be `reflect_value`? I feel like we'd potentially want access to the fields in the standard struct-like fashion.

Reply:
Yeah I think it's fine that the fields aren't public. Reflection provides a way to circumvent Rust's visibility system so it makes sense we can do it here.